src/zalsa_local.rs

```diff
@@ -328,6 +328,30 @@ pub(crate) struct QueryRevisions {
     /// How was this query computed?
     pub(crate) origin: QueryOrigin,
 
+    pub(super) accumulated: Option<Box<AccumulatedMap>>,
```

Comment:
We could probably move `AccumulatedMap` too

I assume you didn't move `accumualted_inputs` because it fits into what otherwise would be padding (similar to verified_final)

Reply:
I'd say lets move accumulated map in there as well, given we already have it lazily allocated (and neither you folks or rust-analyzer seem to use it much at the moment). In fact I still don't see a future where r-a wants to use it tbh 😅